# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1224618553 14400
#      Tue Oct 21 15:49:13 2008 -0400
# Node ID 873ea2c72f16b4f089d721809b419c3c5cc522bb
# Parent  b12a2975bf7e1b32b25fd8ea9912f346076d90ec
quiver3.m: turn hold off after demo

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,14 +1,16 @@
 2008-10-21  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/clf.m: Improve Matlab compatibility.
 
 2008-10-21  John W. Eaton  <jwe@octave.org>
 
+	* plot/quiver3.m: Turn hold off after demo.
+
 	* plot/newplot.m: Delete stray debugging code.
 
 	* plot/gnuplot_drawnow.m: Pass handle to __go_draw_figure__, not
 	struct.  Use get instead of examining struct fields directly.
 	* plot/__go_draw_figure__.m: First arg is now handle, not figure
 	object struct.  Use get instead of examining struct elements
 	directly.  Use allhild instead of looking at children field of
 	figure object struct.
diff --git a/scripts/plot/quiver3.m b/scripts/plot/quiver3.m
--- a/scripts/plot/quiver3.m
+++ b/scripts/plot/quiver3.m
@@ -95,8 +95,9 @@ endfunction
 
 %!demo
 %! [x, y, z] = peaks (25);
 %! surf (x, y, z);
 %! hold on;
 %! [u, v, w] = surfnorm (x, y, z / 10);
 %! h = quiver3 (x, y, z, u, v, w);
 %! set (h, "maxheadsize", 0.33);
+%! hold off;
